examples/demo.js

```diff
@@ -16,11 +15,6 @@ window.map = new Map({
         style:
           'https://vectortiles.geo.admin.ch/styles/ch.swisstopo.leichte-basiskarte_world.vt/style.json',
       },
-      source: new Source({
```

Comment:
If you test on master, you have a button in the bottom right corner of the map. This button shows "c Swisstopo" and is a link to Swisstopo.
With you new system, we miss the link. Is it possible to get it again ? Otherwise, we should be able to manually add attribution again.

Reply:
Setting a custom attribution work the same as before.  I re-add the previous source definition.

I suppose Swisstopo should fix their attribution if they want automatically a good attribution in all of the apps that uses this style